### ISSUE TITLE
loss curve of rar

### ISSUE THREAD

**Issue by ArsenalCheng** [bytedance/1d-tokenizer#88]
Great Work!
I wonder if you could provide a reasonable amount of loss for rar training? I loaded the pretrained weights that you provided and tested for one batch, the loss is usually around 6. If no pretrained weights are provided, the loss is around 7.
